【异常时间】：2019-05-04 21:49:31,903 
【自定义错误】：The ConnectionString property has not been initialized. 
【异常类型】：InvalidOperationException 
【异常信息】：The ConnectionString property has not been initialized. 
【堆栈调用】：   at System.Data.SqlClient.SqlConnection.PermissionDemand()
   at System.Data.SqlClient.SqlConnectionFactory.PermissionDemand(DbConnection outerConnection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Core.Common.Dapper.DbContext.OpenCurrentDbConnection() in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\Core.Common\Dapper\DbContext.cs:line 49
   at Core.Common.Dapper.DbContext.get_Db() in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\Core.Common\Dapper\DbContext.cs:line 35
   at Core.Common.Dapper.DbContext.ExecuteAsync(String sql, Object param, IDbTransaction transaction, Nullable`1 commandTimeout, Nullable`1 commandType) in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\Core.Common\Dapper\DbContext.cs:line 91
   at Core.Repository.UserRepository.UserRegister(LoginVM loginVM) in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\Core.Repository\UserRepository.cs:line 24
   at Core.Services.UserServices.UserRegister(LoginVM loginVM) in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\Core.Services\UserServices.cs:line 28
   at CoreWebApi.Controllers.UserController.UserRegister(LoginVM loginVM) in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\CoreWebApi\Controllers\UserController.cs:line 30
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
--------------------------------------------------------------------
【异常时间】：2019-05-04 21:51:36,619 
【自定义错误】：The ConnectionString property has not been initialized. 
【异常类型】：InvalidOperationException 
【异常信息】：The ConnectionString property has not been initialized. 
【堆栈调用】：   at System.Data.SqlClient.SqlConnection.PermissionDemand()
   at System.Data.SqlClient.SqlConnectionFactory.PermissionDemand(DbConnection outerConnection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Core.Common.Dapper.DbContext.OpenCurrentDbConnection() in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\Core.Common\Dapper\DbContext.cs:line 49
   at Core.Common.Dapper.DbContext.get_Db() in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\Core.Common\Dapper\DbContext.cs:line 35
   at Core.Common.Dapper.DbContext.ExecuteAsync(String sql, Object param, IDbTransaction transaction, Nullable`1 commandTimeout, Nullable`1 commandType) in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\Core.Common\Dapper\DbContext.cs:line 91
   at Core.Repository.UserRepository.UserRegister(LoginVM loginVM) in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\Core.Repository\UserRepository.cs:line 24
   at Core.Services.UserServices.UserRegister(LoginVM loginVM) in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\Core.Services\UserServices.cs:line 28
   at CoreWebApi.Controllers.UserController.UserRegister(LoginVM loginVM) in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\CoreWebApi\Controllers\UserController.cs:line 30
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
--------------------------------------------------------------------
【异常时间】：2019-05-04 21:55:36,800 
【自定义错误】：The ConnectionString property has not been initialized. 
【异常类型】：InvalidOperationException 
【异常信息】：The ConnectionString property has not been initialized. 
【堆栈调用】：   at System.Data.SqlClient.SqlConnection.PermissionDemand()
   at System.Data.SqlClient.SqlConnectionFactory.PermissionDemand(DbConnection outerConnection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Core.Common.Dapper.DbContext.OpenCurrentDbConnection() in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\Core.Common\Dapper\DbContext.cs:line 49
   at Core.Common.Dapper.DbContext.get_Db() in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\Core.Common\Dapper\DbContext.cs:line 35
   at Core.Common.Dapper.DbContext.ExecuteAsync(String sql, Object param, IDbTransaction transaction, Nullable`1 commandTimeout, Nullable`1 commandType) in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\Core.Common\Dapper\DbContext.cs:line 91
   at Core.Repository.UserRepository.UserRegister(LoginVM loginVM) in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\Core.Repository\UserRepository.cs:line 24
   at Core.Services.UserServices.UserRegister(LoginVM loginVM) in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\Core.Services\UserServices.cs:line 28
   at CoreWebApi.Controllers.UserController.UserRegister(LoginVM loginVM) in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\CoreWebApi\Controllers\UserController.cs:line 30
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
--------------------------------------------------------------------
【异常时间】：2019-05-04 21:57:09,905 
【自定义错误】：The ConnectionString property has not been initialized. 
【异常类型】：InvalidOperationException 
【异常信息】：The ConnectionString property has not been initialized. 
【堆栈调用】：   at System.Data.SqlClient.SqlConnection.PermissionDemand()
   at System.Data.SqlClient.SqlConnectionFactory.PermissionDemand(DbConnection outerConnection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Core.Common.Dapper.DbContext.OpenCurrentDbConnection() in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\Core.Common\Dapper\DbContext.cs:line 49
   at Core.Common.Dapper.DbContext.get_Db() in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\Core.Common\Dapper\DbContext.cs:line 35
   at Core.Common.Dapper.DbContext.ExecuteAsync(String sql, Object param, IDbTransaction transaction, Nullable`1 commandTimeout, Nullable`1 commandType) in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\Core.Common\Dapper\DbContext.cs:line 91
   at Core.Repository.UserRepository.UserRegister(LoginVM loginVM) in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\Core.Repository\UserRepository.cs:line 24
   at Core.Services.UserServices.UserRegister(LoginVM loginVM) in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\Core.Services\UserServices.cs:line 28
   at CoreWebApi.Controllers.UserController.UserRegister(LoginVM loginVM) in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\CoreWebApi\Controllers\UserController.cs:line 30
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
--------------------------------------------------------------------
【异常时间】：2019-05-04 21:59:18,415 
【自定义错误】：'UUID' 不是可以识别的 内置函数名称。 
【异常类型】：SqlException 
【异常信息】：'UUID' 不是可以识别的 内置函数名称。 
【堆栈调用】：   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString)
   at System.Data.SqlClient.SqlCommand.CompleteAsyncExecuteReader()
   at System.Data.SqlClient.SqlCommand.EndExecuteNonQueryInternal(IAsyncResult asyncResult)
   at System.Data.SqlClient.SqlCommand.EndExecuteNonQuery(IAsyncResult asyncResult)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at Dapper.SqlMapper.ExecuteImplAsync(IDbConnection cnn, CommandDefinition command, Object param) in C:\projects\dapper\Dapper\SqlMapper.Async.cs:line 678
   at Core.Repository.UserRepository.UserRegister(LoginVM loginVM) in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\Core.Repository\UserRepository.cs:line 29
   at Core.Services.UserServices.UserRegister(LoginVM loginVM) in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\Core.Services\UserServices.cs:line 28
   at CoreWebApi.Controllers.UserController.UserRegister(LoginVM loginVM) in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\CoreWebApi\Controllers\UserController.cs:line 30
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
--------------------------------------------------------------------
【异常时间】：2019-05-04 22:00:25,295 
【自定义错误】：用户 'root' 登录失败。 
【异常类型】：SqlException 
【异常信息】：用户 'root' 登录失败。 
【堆栈调用】：   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Core.Common.Dapper.DbContext.OpenCurrentDbConnection() in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\Core.Common\Dapper\DbContext.cs:line 49
   at Core.Common.Dapper.DbContext.get_Db() in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\Core.Common\Dapper\DbContext.cs:line 35
   at Core.Common.Dapper.DbContext.ExecuteAsync(String sql, Object param, IDbTransaction transaction, Nullable`1 commandTimeout, Nullable`1 commandType) in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\Core.Common\Dapper\DbContext.cs:line 91
   at Core.Repository.UserRepository.UserRegister(LoginVM loginVM) in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\Core.Repository\UserRepository.cs:line 29
   at Core.Services.UserServices.UserRegister(LoginVM loginVM) in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\Core.Services\UserServices.cs:line 28
   at CoreWebApi.Controllers.UserController.UserRegister(LoginVM loginVM) in D:\桌面文件\DotNetCore\.NetCoreWebApi\CoreWebApi\CoreWebApi\Controllers\UserController.cs:line 30
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
--------------------------------------------------------------------
